pkgs/development/python-modules/wgpu-py/default.nix

```
@@ -0,0 +1,157 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  hatchling,
+
+  # preBuild
+  wgpu-native,
+
+  # dependencies
+  cffi,
+  rubicon-objc,
+  sniffio,
+
+  # optional dependency
+  glfw,
+
+  # docs
+  sphinx-rtd-theme,
+  sphinxHook,
+
+  # tests
+  anyio,
+  imageio,
+  numpy,
+  psutil,
+  pytest,
+  ruff,
+  trio,
+
+  # passthru
+  wgpu-py,
+  testers,
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  pname = "wgpu-py";
+  version = "0.21.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "pygfx";
+    repo = pname;
```

```suggestion
    repo = "wdpu-py";
```
Avoid re-using `pname` here.